output/status.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+class Status(Enum):
+    SUCCESS = 0
+    DBNotFound = 1
+    TableNotFound = 2
+    MissingInput = 3
+    FileNotFound = 4
+    WrongInput = 5
+
```

Comment:
Why do you need printName. You can use .name directly.

also print means printing not returning a string.